### DISCUSSION TITLE
Nextra ouside of Vercel

### DISCUSSION THREAD

**Discussion by wotori** [shuding/nextra#162] [Q&A] [answered]
Can I use Nextra outside of Vercel?
I just need to generate static website and host it locally.

**Comment by wotori** [comment, marked as answer]
Now I see - It's possible. 
Just cloned the repo and see how things works here with `yarn dev / start / build`. 

p.s.
why I can`t delete the discussion?

Thanks

**Reply by t0t07** [reply]
Just curious, do you plan to use it with GitLab / GitHub Pages?

**Reply by wotori** [reply]
With GitLab.
As I guess there is no functionality for changing logo between gitlab and github e.t.c.
Wasn't dive deeper about this at this time but some day maybee I will.

Why asking?

**Reply by t0t07** [reply]
@wotori 

I ask because I also tried using Nextra with GitLab Pages, but I ran into several problems. Not sure if you have seen any of these:

- https://github.com/shuding/nextra/issues/142
- https://github.com/shuding/nextra/issues/140

**Reply by wotori** [reply]
Thanks for share your issues. If I'll have solution for any of them i'll share with you.
For now we just keep using the basic interface.

**Reply by vitorgouveia** [reply]
hey uh, i'm having some trouble deploying my nextra website to github pages, how do you set basePath and assetPrefix for nextra? i set mine in next.config.js but it didn't work

**Reply by sebazelonka** [reply]
@VitorGouveia have you solved this?

I've tried some config options, but none of them worked.
